contracts/ERC7432/NftRolesRegistryVault.sol

```diff
@@ -207,21 +224,16 @@ contract NftRolesRegistryVault is IERC7432 {
         revert('NftRolesRegistryVault: role does not exist or sender is not approved');
     }
 
-    /// @notice Checks if an NFT is locked.
+    /// @notice Checks whether an NFT is locked.
     /// @param _tokenAddress The token address.
     /// @param _tokenId The token identifier.
     /// @return True if the NFT is locked.
     function _isLocked(address _tokenAddress, uint256 _tokenId) internal view returns (bool) {
```

Comment:
**suggestion (non-blocking):**  This function checks if the token has an active associated role assignment and not if the token is locked (deposited or frozen for transfers). Maybe we should rename the function for what it's doing